components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -474,11 +474,31 @@ struct AssetTimePrice {
   string price;
 };
 
+struct AssetPriceRequest {
+  CoinType coin_type;
+  string? chain_id;
+  string? address;
+};
+
+enum AssetPriceSource {
+  kCoingecko,
+  kJupiter,
+};
+
+enum Gate3CacheStatus {
+  kHit,
+  kMiss,
+};
+
 struct AssetPrice {
-  string from_asset;
-  string to_asset;
+  CoinType coin_type;
+  string chain_id;
+  string address;
   string price;
-  string asset_timeframe_change;
+  string vs_currency;
+  Gate3CacheStatus cache_status;
```

Comment:
Why frontend needs cache hit/miss status?

Reply:
This is mainly for debugging on the console - never displayed in the UI